src/pages/my-cooperations/MyCooperations.tsx

```diff
@@ -79,13 +76,13 @@ const MyCooperations = () => {
     [filters, page, itemsPerPage, sort]
   )
 
-  const {
-    loading,
-    response,
-    fetchData: getCooperations
-  } = useAxios({
-    service: getMyCooperations,
```

Comment:
Why did you decide to remove the
getMyCoorperations function?
Also with the new PR you can finally use default response without any problems